numpy/core/tests/test_mem_overlap.py

```diff
@@ -94,7 +94,7 @@ def test_overlapping_assignments():
         srcidx = tuple([a[0] for a in ind])
         dstidx = tuple([a[1] for a in ind])
 
-        yield _check_assignment, srcidx, dstidx
```

Comment:
Can we use the `parameterize` decorator here instead?

Reply:
Tried that; doesn't work with nose. I can drop the second commit if you want, but a similar PR was merged previously.

Reply:
Good point, I guess this is fine. Deprecating this style of test feels like a step backwards to me, but that's beyond our control.

Reply:
I added limited support for parameters on nose in `nose_tools` just for some of these cases. It doesn't have the full capabilities of the pytest version, however.

Reply:
Correct. `parametrize` doesn't even replace `yield` because it must be static on collection. I looked at other examples and the same thing was done as here. Complain to the `pytest` folks if you want.